### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by hohaicongthuan**
## _**What's changed:**_

- Improve window title bar buttons (colours, gradient colours).
- Add rules to highlight the scrollbar thumb and buttons when hovering over it.
- Add rules to style hyperlink.

## _**About the scrollbar buttons**_

The scrollbar buttons in Windows 7 behave like this:
- In normal state, the button is shaded totally flat.
- When hovering over the scrollbar (could be the thumb or the track), the button shades will appear.

Here's a [video](https://www.youtube.com/watch?v=H_41nqXx64g) demonstrating it in case you have no clue what I'm talking about.

The problem is I have no idea how to accomplish this. I know that we can set rules to style other elements when hovering another element ([link 1](https://stackoverflow.com/questions/6910049/on-a-css-hover-event-can-i-change-another-divs-styling), [link 2](https://www.geeksforgeeks.org/how-to-affect-other-elements-when-one-element-is-hovered-in-css/)) but I don't know if it's possible for the scrollbar.

### CODE REVIEW

File: gui/_global.scss [hunk 1]
```diff
@@ -29,6 +29,14 @@ h4 {
...
 }
 
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
```
Comment (khang-nd) [inline gui/_global.scss:39]: There's a hyperlink rule available with `_typography.scss`. Could you please help move this over there? Thanks.

File: gui/_global.scss [hunk 2]
```diff
@@ -104,6 +112,28 @@ summary {
...
     }
+
+    &:hover {
+      &:vertical {
+        background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAIAAADpZ+PpAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAADrSURBVChTTc5LboJQGAXguyoCu4ERCzAGlRk7UOwGWIDh0s4M4kxb06RSq/jAB6AxJkJ4lTDrue3AnvyzP+fLId+/yfM8juP7PQmCCOf7B3e+ZD+O40RRVFW12VQUpd3r9U3T2m4OpKoqWZYNwzBZLEqfh0N7NnvfrPcEWlEUWZb9mWF4Ph6D0ylcLbfM5HkeJrhGA2hb15/QXnv+w7RYXsDatjOdvnmrHSnLEizMNE2v11sUXQBCnn98kbquBUGQJAlmq9WB2e3qg4HJdqKkaRql1HGc0WgMcDJ5dd0F24kediZJ8t/ELT69H+8py0CYSIO5AAAAAElFTkSuQmCC")
+            no-repeat center,
+          linear-gradient(
+            to right,
+            var(--button-shade-light-default) 45%,
+            var(--button-shade-light-active) 45%
```
Comment (khang-nd) [inline gui/_global.scss:123]: The color tone is too strong in comparison with the video. Feel free to declare a new variable and use a different color to apply here.

File: gui/_global.scss [hunk 3]
```diff
@@ -104,6 +112,28 @@ summary {
...
+          );
+      }
+  
+      &:horizontal {
+        background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAJCAYAAAALpr0TAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAADcSURBVChTNZBLqoUwEEQrURQUxZGCvy24ACfiityJi7tv8GauQoPxk5tquA2RQ9vVVYk6z9NZaxFFEe77htYazjk8z4MwDIVZ+rourOuKaZrwvi+WZcE8z1BKCbPPCjk4DAO2bRP1OI7wLiL6Mbd7J408z1GWpQwWRYGqqiQG+03TgMu0MacfUN4qANmn8UOv9MjW3sKaSm7iIdOSlziOQ3LScd93aPonSYK6riVLlmVo21aYfVqzND9pmqLrOlGT+76XbcxLZkb19/l3fEP+oF0cx8KMEASBsDEGX2/CgZCHkg+8AAAAAElFTkSuQmCC")
+            no-repeat center,
+          linear-gradient(
+            to bottom,
+            var(--button-shade-light-default) 45%,
+            var(--button-shade-light-active) 45%
```
Comment (khang-nd) [inline gui/_global.scss:133]: Same as above.

**Review by khang-nd — CHANGES_REQUESTED** [review]
Solid work, thanks a bunch mate. 2 points to address though.

Reg. the scrollbar, it might not be possible since there has never been a consolidated approach to style it properly across the browsers, it always feels hacky to do it, so feel free to play around with it as you see fit.

File: gui/_global.scss (hunk 1 above)
Comment (hohaicongthuan) [reply gui/_global.scss:39]: Okay, I'm on it.

File: gui/_global.scss (hunk 2 above)
Comment (hohaicongthuan) [reply gui/_global.scss:123]: Got it.

**Comment by hohaicongthuan** [comment]
Turns out I used the wrong variables. It's all good now.

**Review by khang-nd — APPROVED** [review]
Beautiful, thanks mate.

**Pull request merged**